examples/ansys-aedt-workflow/scripts/DC_Cond_ThermTransient_VariableTimeStep.py

```diff
@@ -0,0 +1,426 @@
+        commandText+=\"nsubst,'''+str(Nsubsteps)+'''\\n\";        
+		commandText+=\"time,\"+endTime.toString()+\"\\n\";
+		commandText+=\"/input,loss,dat\\n\";
+		commandText+=\"save\\n\";
+		var command = myEnvTherm.AddCommandEditor();
+		command.Text = commandText;
+		ds.Script.fillTree();
+		''')
+	fTempFile.close()
+	s2 = os.path.join(Setting.UserDir, "temp.js").replace("\\","\\\\")
+	ThermCompSetup.SendCommand(Command="WB.AppletList.Applet(\"DSApplet\").App.Script.doToolsRunMacro(\""+s2+"\")")
+  
+def Remove_APDL_Command(ThermCompSetup):
+	global Setting
+	fTempFile = open(os.path.join(Setting.UserDir, "temp.js"),"w")
+	fTempFile.write('''
+		var EnvironmentObj = DS.Tree.FirstActiveBranch.Environment;
+		var commandObjs = EnvironmentObj.CommandEditors
+		try{\n
+			var commandObj = commandObjs.Item(1);
+			scriptcode.changeActiveObject(commandObj.ID);
+			DS.Tree.DeleteObject(commandObj.ID);
+		    }\n
+		catch(e)\n
+		    {\n
+		    }\n
+		''')
+	fTempFile.close()
+	s2 = os.path.join(Setting.UserDir, "temp.js").replace("\\","\\\\")
+	ThermCompSetup.SendCommand(Command="WB.AppletList.Applet(\"DSApplet\").App.Script.doToolsRunMacro(\""+s2+"\")")
+
+def CopyFiles(counter):
+	global Setting
+	string = str(counter).zfill(3)
+	string_prev = str(counter-1).zfill(3)
+	if File.Exists(Setting.UserDir+'\\file.rdb'):
+		os.remove(Setting.UserDir+'\\file.rdb')
+	if File.Exists(Setting.UserDir+'\\file.LDHI'):
+		os.remove(Setting.UserDir+'\\file.LDHI')
+	if File.Exists(Setting.UserDir+'\\file.rth'):
+		os.remove(Setting.UserDir+'\\file.rth')
+	if File.Exists(Setting.UserDir+'\\file.r'+string_prev):
+		os.remove(Setting.UserDir+'\\file.r'+string_prev)
+
+	shutil.copyfile(Setting.ThermSystemDir+'\\file.rdb', Setting.UserDir+'\\file.rdb')
+	shutil.copyfile(Setting.ThermSystemDir+'\\file.LDHI', Setting.UserDir+'\\file.LDHI')
+	shutil.copyfile(Setting.ThermSystemDir+'\\file.rth', Setting.UserDir+'\\file.rth')
+	shutil.copyfile(Setting.ThermSystemDir+'\\file.r'+string, Setting.UserDir+'\\file.r'+string)
+	
+def ClearUserDir(counter):
+	global Setting
+	string_prev = str(counter-1).zfill(3)
+	if File.Exists(Setting.UserDir+'\\file.rdb'):
+		os.remove(Setting.UserDir+'\\file.rdb')
+	if File.Exists(Setting.UserDir+'\\file.LDHI'):
+		os.remove(Setting.UserDir+'\\file.LDHI')
+	if File.Exists(Setting.UserDir+'\\file.rth'):
+		os.remove(Setting.UserDir+'\\file.rth')
+	if File.Exists(Setting.UserDir+'\\file.r'+string_prev):
+		os.remove(Setting.UserDir+'\\file.r'+string_prev)
+	if File.Exists(Setting.UserDir+'\\temp.js'):
+		os.remove(Setting.UserDir+'\\temp.js')	
+
+# Variables Initialization
+
+mxwl_project_name = 'TVR14471_V'
+mxwl_design_name  = 'Maxwell3DDesign1'
+mxwl_design_var_name = 'I_pulse'
+
+# read the CSV file path # write a wbjn file to read the csv file path
+# csv_file_name = r"D:\2024_Projects\PyAnsys\pyWorkbenchExample-2\pyWorkbenchExample\ansys-aedt-workflow\10_1000_Pulse.csv"
+input_values = csv.reader(open(csv_file_name))
```

Comment:
You should not hard-code any file path (line 231), or use a file path defined on the client side (line 232). So please replace line 231 and 232 with correct way to accessing data file on server:
(1) Upload the data file (the csv in this case) from the client to the server - you have already done this in main.py.
(2) In the script run by the server (like this script), you assume that the file is in the server's working directory:
```
work_dir = GetServerWorkingDirectory()
csv_file = os.path.join(work_dir, "10_100_Pulse.csv") 
input_values = csv.reader(open(csv_file))
```

Reply:
modified as per the suggestion